Tests/test_file_webp_lossless.py

```diff
@@ -4,37 +4,39 @@
 
 try:
     from PIL import _webp
+    HAVE_WEBP = True
 except ImportError:
-    pass
-    # Skip in setUp()
+    HAVE_WEBP = False
 
 
 class TestFileWebpLossless(PillowTestCase):
 
     def setUp(self):
-        try:
-            from PIL import _webp
-        except:
+        if not HAVE_WEBP:
             self.skipTest('WebP support not installed')
+            return
 
         if (_webp.WebPDecoderVersion() < 0x0200):
             self.skipTest('lossless not included')
 
+        # WebPAnimDecoder only retuns RGBA or RGBX, never RGB
```

Comment:
returns

Reply:
Thanks, fixed the typo (was also in test_file_webp.py too).